src/util/VanDerCorput.cpp

```diff
@@ -116,10 +131,8 @@ VanDerCorput::const_iterator VanDerCorput::end() const
 }
 
 //=============================================================================
-VanDerCorput::const_iterator::const_iterator(const VanDerCorput *seq)
-    : mSeq(seq)
-    , mN(-1)
-    , mFinalIter(false)
+VanDerCorput::const_iterator::const_iterator(const VanDerCorput* seq)
+  : mSeq(seq), mN(-1), mFinalIter(false)
```

Comment:
Also, this is exactly why I prefer to spend time hand formatting the code to pass the **check** rather than spending too much time trying to make the **format** operation idempotent: there are going to be many issues like this where we want something that the formatter won't do, and the other issue with clang 3.9's formatter just churning the code for no reason.